README-Networking.md

```diff
@@ -33,9 +39,32 @@ CNI-Genie is a CNI plugin that enables Kubernetes to simultaneously have access
   - Service Discovery
     - micro DNS at each node, DNS lookups always local
   - Multicast Networking - UDP multcast even if underlying network doesn't support it
-  - 
+
 **Flannel**
 
+Try it out: `kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml`
+  - Focused on networking. For network policy, other projects such as Calico can be used
+  -  Main advantage of this model is that it removes the port mapping complexities that come from sharing a single host IP
+  - Does not control how containers are networked to the host, only how the traffic is transported between hosts
+  - Service Discovery
+    - Runs a small, single binary agent called `flanneld` on each host, and is responsible for allocating a subnet lease to each host out of a larger, preconfigured address space
+    - uses either the Kubernetes API or etcd directly to store the network configuration, the allocated subnets, and any auxiliary data (such as the host's public IP)
+
 **Calico**
 
-**Romana**
\ No newline at end of file
+Try it out: `kubectl apply -f \
+https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml`
+
+  - Enables networking and network policy
+  - Secure component communication: can configure etcd to encrypt its communications with TLS and require clients to present certificates signed by the etcd certificate authority.
```

Comment:
is this the main way people get around this limitation? Check to see if the newer versions of kubernetes already do this thus making this obsolete.